### PR TITLE
Update installation guide to use charmhub

### PR DESCRIPTION
**Pull request by heitorPB**
The latest bundle pulls the charms from Charmhub, without needing to
have to build the charms locally.

See https://github.com/omnivector-solutions/slurm-bundles/pull/17

### CODE REVIEW

File: installation.rst [hunk 1]
```diff
@@ -215,20 +200,27 @@ basic Slurm cluster:
...
 .. code-block:: bash
 
    $ juju deploy ./slurm-core/bundle.yaml \
                  --overlay ./slurm-core/clouds/lxd.yaml \
                  --overlay ./slurm-core/series/focal.yaml
 
-Juju will then create the applications, configurations and LXD containers
-described in the respective files, which will in turn define the contents of
-the model.
+Juju will then dowload the charms from Charmhub, create the applications,
```
Comment (matheushent) [inline installation.rst:221]: "download" instead of "dowload"

**Review by matheushent — CHANGES_REQUESTED** [review]

File: installation.rst [hunk 2]
```diff
@@ -296,3 +288,29 @@ After setting the node up, to bring it back you need to run a Juju *action*:
 
 Please refer to our :ref:`operations` section for detailed instructions on how
 to manage the cluster.
+
+Local charms
```
Comment (jamesbeedy) [inline installation.rst:292]: want to move this to the development docs?

Comment (heitorPB) [reply installation.rst:292]: I will do that in the next PR. This one here is not aware of that document yet.

File: installation.rst [hunk 3]
```diff
@@ -215,20 +200,27 @@ basic Slurm cluster:
...
    $ juju deploy ./slurm-core/bundle.yaml \
                  --overlay ./slurm-core/clouds/lxd.yaml \
                  --overlay ./slurm-core/series/focal.yaml
 
-Juju will then create the applications, configurations and LXD containers
-described in the respective files, which will in turn define the contents of
-the model.
+Juju will then download the charms from Charmhub, create the applications,
+configurations and LXD containers described in the respective files, which will
+in turn define the contents of the model.
```
Comment (jamesbeedy) [inline installation.rst:223]:
```suggestion
comprise the model.
```

**Comment by jamesbeedy** [comment]
Nice work - LGTM 👍

**Review by matheushent — APPROVED** [review]

**Pull request merged**